CSS/ex_16/contact.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact | It's Just Jack</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Homemade+Apple&display=swap');
+        header {
+            text-align: center ;
+            color:rgb(241, 77, 55) ;
+        }
+        h1 {
+            text-align: center;
+        }
+
+        h2 {
+            font-size: 30px ;
+            font-family: 'Homemade Apple', cursive ;
+        }
+        
+        /* Flexbox navigation goes below here */
+        .wrapper {
+            border-top: 5px dotted coral ;
+            border-bottom: 5px dotted coral ;
+            max-width: 200px ;
+            margin: auto ;
+            padding: 10px ;
+        }
+
+        .flex-nav ul {
+            list-style: none ;
+            margin: 0 ;
+            padding: 0 ;
+            display: flex ;
+        }
+
+        .flex-nav li {
+            flex: 1 ;
+        }
+
+        /*Inset border */
+        hr {
+            height: 10px ;
+	        border: 0 ;
+            box-shadow: 0 10px 10px -10px #8c8b8b inset ;
+            margin-top: 30px;
+        }
+
+        /* Special quote text */
+        .p-quote {
+            font-family: 'Homemade Apple', cursive ;
+            font-size: 12px;
+            margin-left: 30px ;
+        }
+
+        /* Form styles*/
+        .form-name {
+            margin-left: 12px;
+            padding: 12px
+        }
+        .form-email {
+            margin-left: 12px;
+            padding: 12px
+        }
+        .form-message {
+            margin-left: 12px;
+            padding: 12px
+        }
+        .form-submit {
+            margin-left: 12px;
+            padding: 12px
+        }
+    </style>
+</head>
+
+<body>
+    <header>
+        <img src="https://pbs.twimg.com/profile_images/1169816682841264128/iaYgus-J.jpg" alt="Photo">
+        <h2>Jack's Best Quotes</h2>
+        <div class="wrapper">
+            <nav class="flex-nav">
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About</a></li>
+                <li><a href="contact.html">Contact</a></li>
+            </ul>
+            </nav>
+        </div>
+    </header>
+
+    <hr />
+
+    <main>
+    <h1>Contact</h1>
+    <p>Looking for a pirate to sail among the tides of the ocean, or to hunt treasure? You know, I am a man of honour. Fill in the form below, and I shall give you a "good price".</p>
+    
+    <form action="contact-success.html" method="get">
+        <div class="form-name">
+            Name:<input type="name" name="name" placeholder="Name or Alias" required> <br />
+        </div>
```

Comment:
Really nice that you used wrapper div around input field. That's a good practice. You will find it in Css Frameworks like Bootstrap and others.